components/Carousel/Carousel.js

```diff
@@ -0,0 +1,9 @@
+class Carousel {
+  constructor (element) {
+    this.element = element;
+    this.figureImg = this.element.querySelector('.carousel-image');
+    this.figureCap = this.element.querySelector('.carousel-caption');
+  }
+}
+
+const test = new Carousel(document.querySelector('.carousel'));
```

Comment:
I'm glad to see that you were starting to build out your own carousel! hopefully what we went over in afterhours last night gave you everything else you need :smile: